samples/java-customer-registry/src/main/java/customer/CustomerValueEntityHandler.java

```diff
@@ -28,70 +28,59 @@
 import com.google.protobuf.InvalidProtocolBufferException;
 import customer.api.CustomerApi;
 import customer.domain.CustomerDomain;
-import scala.None$;
-import scala.Option;
 import scalapb.UnknownFieldSet;
+import java.util.Optional;
 
 /**
  * This class would be generated by codegen and should be treated mostly as an implementation detail
  * by the user.
  */
 public class CustomerValueEntityHandler implements ValueEntityHandler {
+
   final CustomerValueEntity entity;
-  final AnySupport anySupport;
 
   public static final Descriptors.ServiceDescriptor serviceDescriptor =
       CustomerApi.getDescriptor().findServiceByName("CustomerService");
   public static final String entityType = "customers";
 
-  CustomerValueEntityHandler(CustomerValueEntity entity) {
-    this.entity = entity;
-    this.anySupport =
```

Comment:
very nice, indeed no longer needed because we no longer allow updating the state via the context :+1: